docs/custom-commands.md

```diff
@@ -6,15 +6,15 @@ It is possible to extend fin with custom commands (per project).
 
     File `.docksal/commands/updb` (Notice **no extension**. Script name should match command name)
 
-    ```bash
+```bash
```

Comment:
This should be reverted to what it was. In lists anything in-between has to be indented with 4 spaces.